tools/querytee/response_comparator.go

```diff
@@ -158,12 +161,15 @@ func compareVector(expectedRaw, actualRaw json.RawMessage, opts SampleComparison
 		metricFingerprintToIndexMap[actualMetric.Metric.Fingerprint()] = i
 	}
 
+	missingMetrics := make([]model.Metric, 0)
 	for _, expectedMetric := range expected {
 		actualMetricIndex, ok := metricFingerprintToIndexMap[expectedMetric.Metric.Fingerprint()]
 		if !ok {
-			return fmt.Errorf("expected metric %s missing from actual response", expectedMetric.Metric)
+			missingMetrics = append(missingMetrics, expectedMetric.Metric)
```

Comment:
Consider adding a test to ensure that the expected number of missing series shows up in the metric.
See https://pkg.go.dev/github.com/prometheus/client_golang/prometheus/testutil

Reply:
Nice. There was actually an issue with the buckets.